subsys/bluetooth/host/conn.c

```diff
@@ -139,7 +139,7 @@ void frag_destroy(struct net_buf *frag)
 	/* allow next view to be allocated (and unlock the parent buf) */
 	bt_buf_destroy_view(frag, &get_frag_md(frag)->view_meta);
 
-	LOG_DBG("");
+	LOG_DBG("%s", __func__);
```

Comment:
What was the warning here? Empty log_dbg strings are fairly common because it logs the name of the function by default. I see at least 500 cases of this in the tree

Reply:
Huh. 

This is going to be a growing pain if we don't fix the root cause. We don't want to apply this fix to all 500+ cases in the codebase. 

I would be OK with approving this as is, given that it would otherwise block this PR, but we would need a better solution for `LOG_DBG("")` for the `unit_testing` board

Reply:
hang on, gcc and clang sets `-Wno-format-zero-length`, let's set it for unit test as well

Reply:
ok added a flag, not sure it's the right/best location but I guess the buildsystem folks will point that out